File: engine/engine-model/src/main/scala/tools/packet/statusinfo/ShowQuestForfeit.scala
package tools.packet.statusinfo

import net.opcodes.SendOpcode
import tools.packet.PacketInput

class ShowQuestForfeit(private var _questId: Short) extends PacketInput {
  def questId: Short = _questId

  override def opcode(): SendOpcode = SendOpcode.SHOW_STATUS_INFO
}

File: engine/engine-model/src/main/java/tools/packet/statusinfo/ShowQuestForfeit.java
package tools.packet.statusinfo;

import net.opcodes.SendOpcode;
import tools.packet.PacketInput;

public record ShowQuestForfeit(Short questId) implements PacketInput {
   @Override
   public SendOpcode opcode() {
      return SendOpcode.SHOW_STATUS_INFO;
   }
}